src/components/Select/index.js

```diff
@@ -115,43 +115,28 @@ export const Select = React.forwardRef(function Select(
               'contacto-multi-select-modal',
               maxTagModalTitle ? '' : 'contacto-multi-select-modal--no-title',
             ].join(' ')}
-            title={maxTagModalTitle}
+            title={'Hello'}
             visible={showMaxTagModal}
             onCancel={() => setShowMaxTagModal(false)}
             cancelButtonProps={null}
           >
             <div className="contacto-multi-select-wrapper">
               <AntSelect
+                size="large"
                 className="contacto-multi-select"
                 showArrow={false}
                 listHeight={listHeight || 220}
-                tagRender={() => {}}
+                placeholder={placeholder}
+                tagRender={(tagProps) => (
+                  <Tag type="select" disableUppercase closeIcon {...tagProps} />
+                )}
                 dropdownClassName={['sg contacto-select-listbox', dropdownClassName].join(' ')}
                 mode="multiple"
                 value={localValue}
                 onChange={onValueChange}
-                style={{ '--placeHolderText': `"${placeholder}"` }}
+                maxTagCount={null}
```

Comment:
Can we make this a prop? 
And default it to `null` if not found?

Reply:
We don't want it to be anything except null, a better way is to just not pass it, and remove {...props}

Done ✅